src/hydrabus/hydrabus_mode_smartcard.c

```diff
@@ -486,6 +500,29 @@ static uint32_t read(t_hydra_console *con, uint8_t *rx_data, uint8_t nb_data)
 	return status;
 }
 
+static uint32_t tread(t_hydra_console *con, uint8_t *rx_data, uint8_t nb_data)
+{
+	uint8_t nb_rdata, i;
+	mode_config_proto_t* proto = &con->mode->proto;
+
+	nb_rdata = bsp_smartcard_read_u8_timeout(proto->dev_num, rx_data, nb_data, proto->config.smartcard.dev_timeout * 10);
```

Comment:
Avoid hard coded value "proto->config.smartcard.dev_timeout * 10" it shall be TIME_MS2I(proto->config.smartcard.dev_timeout) (to be checked as the unit is expected to be in ms)

Reply:
done.